rsocket/RSocketClient.cpp

```diff
@@ -47,6 +47,12 @@ const std::shared_ptr<RSocketRequester>& RSocketClient::getRequester() const {
 folly::Future<folly::Unit> RSocketClient::resume() {
   VLOG(2) << "Resuming connection";
 
+  if (disconnectBaton_) {
```

Comment:
this is not thread safe (and all the other places where we use the Baton). Why do we need it anyways?

Reply:
I was just about to remove the baton's. It was for temporarily testing.